src/IO/WriteHelpers.h

```diff
@@ -380,6 +380,162 @@ void writeAnyEscapedString(const char * begin, const char * end, WriteBuffer & b
     }
 }
 
+template <char quote_character, bool escape_quote_with_quote = false, bool escape_backslash_with_backslash = true>
+void writeAnyMarkdownEscapedString(const char * begin, const char * end, WriteBuffer & buf)
+{
+    const char * pos = begin;
+    while (true)
+    {
+        const char * next_pos = find_first_symbols_markdown<
+            '\b',
+            '\f',
+            '\n',
```

Comment:
Cosmetics: Could we have all special characters on the same line like in l. 324?